From 03ced091cd830fe4d16ad1596228bc035e4b961c Mon Sep 17 00:00:00 2001
From: Mika Laitio <lamikr@gmail.com>
Date: Wed, 24 Jan 2024 14:53:02 -0800
Subject: [PATCH 1/3] disable prrte docs generation

Disable prrte doc building to avoid follwing error
if python sphinx is not installed available:

make[5]: Leaving directory '/home/lamikr/own/rocm/src/sdk/rocm_sdk_builder_571/builddir/011_07_openmpi/3rd-party/prrte/src/docs/prrte-rst-content'
Making all in show-help-files
make[5]: Entering directory '/home/lamikr/own/rocm/src/sdk/rocm_sdk_builder_571/builddir/011_07_openmpi/3rd-party/prrte/src/docs/show-help-files'
  GENERATE text files
python3: can't open file '/home/lamikr/own/rocm/src/sdk/rocm_sdk_builder_571/src_projects/openmpi/3rd-party/prrte/src/docs/show-help-files/build-dummy-ini-files.py': [Errno 2] No such file or directory
touch: cannot touch '_build/text/index.txt': No such file or directory
make[5]: *** [Makefile:961: _build/text/index.txt] Error 1
make[5]: Leaving directory '/home/lamikr/own/rocm/src/sdk/rocm_sdk_builder_571/builddir/011_07_openmpi/3rd-party/prrte/src/docs/show-help-files'
make[4]: *** [Makefile:687: all-recursive] Error 1
make[4]: Leaving directory '/home/lamikr/own/rocm/src/sdk/rocm_sdk_builder_571/builddir/011_07_openmpi/3rd-party/prrte/src/docs'
make[3]: *** [Makefile:1706: all-recursive] Error 1
make[3]: Leaving directory '/home/lamikr/own/rocm/src/sdk/rocm_sdk_builder_571/builddir/011_07_openmpi/3rd-party/prrte/src'
make[2]: *** [Makefile:802: all-recursive] Error 1
make[2]: Leaving directory '/home/lamikr/own/rocm/src/sdk/rocm_sdk_builder_571/builddir/011_07_openmpi/3rd-party/prrte'
make[1]: *** [Makefile:1407: all-recursive] Error 1
make[1]: Leaving directory '/home/lamikr/own/rocm/src/sdk/rocm_sdk_builder_571/builddir/011_07_openmpi/3rd-party'
make: *** [Makefile:1534: all-recursive] Error 1
build failed: openmpi

Signed-off-by: Mika Laitio <lamikr@gmail.com>
---
 3rd-party/prrte/src/Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/3rd-party/prrte/src/Makefile.am b/3rd-party/prrte/src/Makefile.am
index a623100..f98c89e 100644
--- a/3rd-party/prrte/src/Makefile.am
+++ b/3rd-party/prrte/src/Makefile.am
@@ -27,7 +27,6 @@
 #
 
 SUBDIRS = \
-	docs \
 	etc \
 	util \
 	$(MCA_prte_FRAMEWORKS_SUBDIRS) \
@@ -36,7 +35,6 @@ SUBDIRS = \
 	$(MCA_prte_FRAMEWORK_COMPONENT_DSO_SUBDIRS)
 
 DIST_SUBDIRS = \
-	docs \
 	etc \
 	util \
 	$(MCA_prte_FRAMEWORKS_SUBDIRS) \
-- 
2.45.1

